### DISCUSSION TITLE
Using Multiple devices

### DISCUSSION THREAD

**Discussion by daynial132** [ActivityWatch/activitywatch#1358] [Q&A]
This app is great. I am using it for more then 4 months & its awesome. I have use last version, and add multiple devices and need sync all devices on a single server but not able to do now 
 
I configure it through .ini files of server and client

what can i do now ??

**Comment by QazCetelic** [comment]
For anyone interested, there currently is a "Use multidevice query" feature that fetches data from all devices. You can then use this with a single ActivityWatch host to have a single overview.
